test/fixtures/keys/Makefile

```diff
@@ -67,7 +183,7 @@ fake-startcom-root-cert.pem: fake-startcom-root.cnf \
 	openssl req -new -x509 -days 99999 -config \
 	fake-startcom-root.cnf -key fake-startcom-root-key.pem -out \
 	fake-startcom-root-cert.pem
-	echo '01' > fake-startcom-root-serial
+	echo '03' > fake-startcom-root-serial
```

Comment:
For my edification: this changes because...? (new timestamp and OU?)

Reply:
accident, because I was having trouble regenerating. fixed, thanks.